tools/notification-configuration/notification-creator/NotificationConfigurator.cs

```diff
@@ -275,6 +292,39 @@ private async Task SyncTeamWithCodeOwnerFile(BuildDefinition pipeline, WebApiTea
             }
         }
 
+        private CodeownersEntry GetMatchingCodeownersEntry(YamlProcess process, List<CodeownersEntry> codeownersEntries)
+        {
+            // See comment on PipelineOwnerSyntheticFileName to understand why the pipelineOwnerFile
+            // is not simply process.YamlFilename.
+            string pipelineOwnerFile = GetSiblingFilePath(
+                targetPath: process.YamlFilename,
+                siblingFileName: PipelineOwnerSyntheticFileName);
```

Comment:
FYI this `GetSiblingFilePath` invocation is the actual behavior change. Previously the code was equivalent to:

`string pipelineOwnerFile = process.YamlFilename;`